src/audio/module_adapter/module/volume/volume_hifi4_with_peakvol.c

```diff
@@ -0,0 +1,379 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2022 Intel Corporation. All rights reserved.
```

Comment:
2022 --> 2023

Reply:
I didn't rewrite this document, so I don't think need to change like this.